factory/consensus/consensusComponents.go

```diff
@@ -641,6 +655,19 @@ func (ccf *consensusComponentsFactory) createConsensusTopic(cc *consensusCompone
 	return ccf.networkComponents.NetworkMessenger().RegisterMessageProcessor(cc.consensusTopic, common.DefaultInterceptorsIdentifier, cc.worker)
 }
 
+func (ccf *consensusComponentsFactory) createPeerBlacklistHandler() (consensus.PeerBlacklistHandler, error) {
```

Comment:
add unit test?

Reply:
this is covered in principle in the main create unit test